DATA/production/qc-workflow.sh

```diff
@@ -157,8 +157,13 @@ elif [[ -z ${QC_JSON_FROM_OUTSIDE:-} ]]; then
     [[ -z "${QC_JSON_FT0:-}" ]] && QC_JSON_FT0=$O2DPG_ROOT/DATA/production/qc-async/ft0.json
     [[ -z "${QC_JSON_FV0:-}" ]] && QC_JSON_FV0=$O2DPG_ROOT/DATA/production/qc-async/fv0.json
     [[ -z "${QC_JSON_FDD:-}" ]] && QC_JSON_FDD=$O2DPG_ROOT/DATA/production/qc-async/fdd.json
-    [[ -z "${QC_JSON_EMC:-}" ]] && QC_JSON_EMC=$O2DPG_ROOT/DATA/production/qc-async/emc.json
     [[ -z "${QC_JSON_MID:-}" ]] && QC_JSON_MID=$O2DPG_ROOT/DATA/production/qc-async/mid.json
+    if [[ -z "${QC_JSON_EMC:-}" ]]; then
+      if [[ "$BEAMTYPE" == "PbPb" ]]; then
```

Comment:
Should be fine to have this check here for async. @mfasDa , please, pay attention as soon as we start cpass over these data.

Reply:
@chiarazampolli In the special reconstruction of the PbPb runs in unstable beam I still see the multiplicity ranges for pp applied, not the ones for PbPb (i.e. number of cell distribution cut at 1000 while it can go as high as 6000 in central PbPb). Either an old version of O2DPG was used or the check for the beamtype does not work in the async mode (corresponding check in the sync QC works fine).

Reply:
Hello, 
We used a previous version of O2DPG, since the reco was requested with urgency on Friday by FT0. 
Cheers,
Chiara

Reply:
Hi @chiarazampolli ,

OK, then it is clear. We will check for the next runs again.

Cheers

Markus

Reply:
Hello @mfasDa , 
Please, add a comment in the production request too. 
If needed we can of course redo, but these data are now needed, I guess, so if we delete the production, it might be an issue.
Cheers,
Chiara